CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## 16.11.0
+
+- Changed: deprecation warning to link to `https://stylelint.io/awesome-stylelint#formatters` ([#8115](https://github.com/stylelint/stylelint/pull/8115)) ([@ryo-manba](https://github.com/ryo-manba)).
```

Comment:
Should we also use "Fixed:" here? We typically use "Changed" (along with "Removed") as triggers for major releases:

- `major`: "Changed" and "Removed"
- `minor`: "Added" and "Deprecated"
- `patch`: "Fixed" and "Security"

https://keepachangelog.com/en/1.1.0/

Reply:
Agreed with using "Fixed" instead of "Changed" 👍🏼 

Also, I think it's helpful for maintainers to add that convention to our guide:
https://github.com/stylelint/stylelint/blob/3bf048d2e0c7d67841e0fcc7298788ae89c5a7e9/docs/maintainer-guide/pull-requests.md?plain=1#L22-L24

Reply:
@jeddy3 just a clarification, when something which was undocumented gets removed/changed should we inform the users in the changelog?
cf https://github.com/stylelint/stylelint/pull/8106#discussion_r1836824152

Reply:
We typically only mention changes to our public and documented API in our Changelog. The only time we didn't was when we refactored to `.cjs` and `.mjs` files because the change was wide-reaching and known to impact plugin authors who imported our internal utilities.